htsget_server/operations.py

```diff
@@ -340,7 +340,14 @@ def get_data(id, referenceName=None, format=None, start=None, end=None):
                 index_filename=index_file
             )
 
-        file_out = AlignmentFile(ntf.name, 'w', header=file_in.header)
+        if format == "bam":
```

Comment:
this isn't needed if it defaults to `bam` right?

Reply:
That's a good point.

Reply:
Fixed.